packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -10,7 +10,7 @@ export let message: string;
 /** Function called when the close button is clicked */
 export let dismiss: () => void;
 /** The action performed on the button */
-export let action: { text: string; handler: () => unknown } | undefined =
+export let action: { text: string; handler: () => void } | undefined =
```

Comment:
[Nit] I find `unknown` to be a more useful return type in signatures for interaction handler, was there a reason for this change?

Reply:
I wanted it to be more consistent with the dismiss function which has a return type of void

Reply:
But since it's an interaction handler for actions I can make it `unknown`